test/test_samples_osw/base/results_output.csv

```diff
@@ -1,58 +1,58 @@
-project_national-002.osw,4.9,0.3,0.2,0.0,-9.5,0.6,0.0,0.0,0.0,0.0,16.9,0.1,0.0,0.0,9.3,0.2,5.2,0.6,74.6,0.0,0.0,74.6,20.0,603.8,11.0,0.0,0.2,61.2,0.1,34.3,491.4,4505.8,0.3,81.6,0.0,167.4,0.0,0.0,1780.2,18.4,320.0,28.7,5.3,0.0,962.5,950.2,0.0,803.1,4500.0,0.0,0.0,0.0,0.0,-0.0,11.0,0.0,1.6,480.4,0.0,0.0,0.0,0.0,6.2,-0.7,1376.0,2176.0,2752.0,0.0,0.0,0.0,2.1,0.0,0.4,2.1,2375.0,0.0,5404.9,8433.5,0.0,true,0.9,1.0,0.0,0.0,0.5,0.0,-0.0,2.8,27.7,0.0,0.0,0.0,26.5,0.0,0.0,0.0,2.3,217.6,21.2,0.5,0.0,0.0,1.1,-0.2,0.0,0.0,0.0,0.0,0.0,0.0,1538.4,18.4,28.8,40.0,17.7,-0.4,0.3,0.4,0.2,0.3,15212.4,79.1,0.0,271.8,0.0,0.0,0.0,0.0,2394.9,2810.0,406.7,312.4,0.0
-project_national-011.osw,0.9,-0.0,0.1,0.0,-3.0,0.0,0.0,0.0,0.0,0.0,2.0,0.0,0.0,0.0,0.9,0.0,0.6,0.0,9.6,0.0,0.2,9.6,20.0,283.2,11.0,75.5,0.2,51.0,0.1,28.6,65.4,1197.5,0.0,0.0,0.0,79.2,1.6,480.4,0.0,0.3,0.0,0.0,0.2,0.0,0.0,0.0,0.0,502.4,1885.9,0.0,1753.6,0.0,0.0,-0.0,11.0,0.0,1.6,480.4,0.0,0.0,0.0,0.0,2.6,-0.0,885.0,885.0,885.0,0.0,0.0,0.0,0.0,0.0,0.5,0.0,229.0,0.0,2701.9,5270.1,0.0,true,-1.9,0.1,0.0,0.0,0.2,0.0,0.0,2.3,23.1,23.2,0.0,0.0,0.4,0.0,0.0,165.7,1.4,132.3,1.2,0.0,0.0,0.0,0.7,-0.0,0.0,0.0,0.0,0.0,0.0,0.0,989.5,9.2,18.0,40.0,5.2,-0.0,0.4,0.1,0.2,2.2,6622.2,57.1,0.0,344.7,0.0,0.0,0.0,0.0,993.4,1116.3,993.4,145.4,0.0
-project_national-014.osw,6.3,9.3,0.1,0.1,5.5,7.3,0.0,0.0,0.0,0.0,8.8,7.6,0.0,0.0,5.3,6.4,0.6,1.3,37.3,0.1,0.1,37.4,20.0,696.3,5.5,0.0,0.1,33.4,0.1,18.7,268.1,5648.4,0.2,44.6,0.0,194.8,2.5,727.6,0.0,0.0,320.0,33.4,87.9,0.0,0.0,0.0,0.0,934.4,2369.3,0.0,0.0,0.0,1.7,-0.0,11.0,0.0,1.5,434.3,0.0,0.0,0.0,0.0,2.5,-2.1,2176.0,2176.0,2752.0,0.0,0.0,0.0,46.5,0.0,0.1,46.5,0.0,1.0,11677.8,11693.9,0.0,true,3.2,19.7,0.0,0.0,0.3,0.0,-0.3,1.5,15.2,0.0,0.0,0.0,590.1,0.0,14.4,0.0,1.0,91.8,1.0,-0.7,0.0,0.0,0.5,-0.4,0.0,0.0,0.0,0.0,0.0,0.0,3076.8,39.9,39.9,30.0,3.3,-1.7,0.5,0.5,0.4,0.3,11133.2,109.1,0.0,711.5,0.0,0.0,0.0,0.0,1657.5,2263.8,684.8,121.1,0.0
-project_national-046.osw,0.0,5.5,0.0,0.0,0.0,4.7,0.0,0.0,0.0,0.0,0.0,4.4,0.0,0.0,0.0,4.4,0.0,0.8,0.0,0.0,0.0,0.0,20.0,739.0,11.0,104.7,0.2,51.0,0.1,28.6,409.5,0.0,0.3,90.9,0.0,238.4,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,1097.3,2387.3,0.0,0.0,0.0,0.0,-0.0,11.0,0.0,1.5,434.3,0.0,0.0,52.7,0.0,0.0,-1.4,1331.5,2663.0,2663.0,0.0,0.0,0.0,17.3,-0.2,-1.4,17.1,4380.0,19.0,0.0,13283.7,0.0,true,0.0,3.1,0.1,0.0,0.0,0.0,-0.2,2.3,23.1,0.0,0.0,0.0,253.0,0.0,0.0,0.0,1.2,115.4,0.0,-0.7,0.0,0.0,0.0,-0.4,0.0,0.0,0.0,0.0,0.0,0.0,1488.7,0.0,45.3,40.0,0.0,-3.0,0.4,0.3,0.2,0.2,4916.6,55.9,0.0,391.4,0.0,0.0,0.0,0.0,2436.9,2621.9,609.2,289.5,0.0
```

Comment:
Hmm, why are we seeing results change for every sample when very few samples have finished attics. Do you just need to pull develop in again?

Reply:
I pulled in the latest. You can see that nothing changes. This is because we don't model finished attics (sampled finished attics are modeled as vented unfinished attics).